Library/Homebrew/cask/lib/hbc/system_command.rb

```diff
@@ -47,7 +47,8 @@ def initialize(executable, args: [], sudo: false, input: [], print_stdout: false
       @must_succeed = must_succeed
       options.extend(HashValidator).assert_valid_keys(:chdir)
       @options = options
-      @path = path
+      @env = { path: ENV["PATH"] }
+      yield(@env) if block_given?
```

Comment:
Instead of yielding this I would make this another named parameter and merge it with `{ path: ENV["PATH"] }`.

Reply:
Thanks for the feedback. Changed.